components/features/bug-reporting/screenshot-capture.tsx

```
@@ -1 +1,476 @@
+			ctx.drawImage(annotationCanvas, 0, 0);
+
+			// Convert to blob
+			compositeCanvas.toBlob((blob) => {
+				if (blob) {
+					const file = new File([blob], "screenshot.png", { type: "image/png" });
+					onScreenshotCapture?.(file, annotations);
+				}
+			}, "image/png");
+		};
+		img.src = screenshot;
+	}, [screenshot, annotations, onScreenshotCapture]);
```

Biome (2.1.2)

[error] This async function lacks an await expression.

Remove this async modifier, or add an await expression in the function.

Async functions without await expressions may not need to be declared async.

(lint/suspicious/useAwait)